install_config/install/deploy_router.adoc

```diff
@@ -265,6 +265,217 @@ $ iptables -A INPUT -p tcp --dport 10443 -j ACCEPT
+ilustrates how `oadm router`, `oc env` and `oc scale`
+work together to make a router shard.
+
+====
+[source,bash]
+----
+#!/bin/bash
+# Usage: mkshard ID SELECTION-EXPRESSION
+id=$1
+sel="$2"
+router=router-shard-$id           //<1>
+oadm router $router --replicas=0  //<2>
+dc=dc/router-shard-$id            //<3>
+oc env   $dc ROUTE_LABELS="$sel"  //<4>
+oc scale $dc --replicas=3         //<5>
+exit $?
+----
+<1> The created router has name `router-shard-<id>`.
+<2> Specify no scaling for now.
+<3> The deployment configuration for the router.
+<4> Set the selection expression using `oc env`.
+    The selection expression is the value of
+    the `ROUTE_LABELS` environment variable.
+<5> Scale it up.
+====
+
+Running *_mkshard_* several times creates several routers:
+
+[options="header",cols="2,3,2"]
+|===
+|Router |Selection Expression |Routes
+
+|`router-shard-1`
+|`sla=high`
+|`a`, `l` -- `s`
+
+|`router-shard-2`
+|`geo=west`
+|`b`, `t` -- `z`
+
+|`router-shard-3`
+|`dept=dev`
+|`g` -- `k`
+
+|===
+
+
+[[modifying-router-shards]]
+=== Modifying Router Shards
+
+Because a router shard is a construct
+link:../../architecture/core_concepts/routes.html#router-sharding[based on labels],
+you can modify either the labels (via
+link:../../cli_reference/basic_cli_operations.html#application-modification-cli-operations[`oc label`])
+or the selection expression.
+
+This section extends the example started in the
+link:#creating-router-shards[Creating Router Shards] section,
+demonstrating how to change the selection expression.
+
+Here is a convenience script *_modshard_* that modifies
+an existing router to use a new selection expression:
+
+====
+[source,bash]
+----
+#!/bin/bash
+# Usage: modshard ID SELECTION-EXPRESSION...
+id=$1
+shift
+router=router-shard-$id       //<1>
+dc=dc/$router                 //<2>
+oc scale $dc --replicas=0     //<3>
+oc env   $dc "$@"             //<4>
+oc scale $dc --replicas=3     //<5>
+exit $?
+----
+<1> The modified router has name `router-shard-<id>`.
+<2> The deployment configuration where the modifications occur.
+<3> Scale it down.
+<4> Set the new selection expression using `oc env`.
+    Unlike `mkshard` from the
+    link:#creating-router-shards[Creating Router Shards]
+    section, the selection expression specified as the
+    non-`ID` arguments to `modshard` must include the
+    environment variable name as well as its value.
+<5> Scale it back up.
+====
+
+For example, to expand the department for `router-shard-3`
+to include `ops` as well as `dev`:
+
+----
+$ modshard 3 ROUTE_LABELS='dept in (dev, ops)'
+----
+
+The result is that `router-shard-3` now selects routes `g` -- `s`
+(the combined sets of `g` -- `k` and `l` -- `s`).
+
+This example takes into account that
+there are only three departments in this example scenario,
+and specifies a department to leave out of the shard,
+thus achieving the same result as the preceding example:
+
+----
+$ modshard 3 ROUTE_LABELS='dept != finanace'
+----
```

Comment:
@bfallonf I don't think so.  The "language feature" highlighted here is the `!=` operator, which i believe means "does not equal".  So:

    ROUTE_LABELS='dept=finance'

is valid, but selects those routes key `dept` *equal to* `finance`, which is opposite of the preceding example (and contradicts the example's intro blurb).  Thus, to complete the logical chain, the desired operator should be flipped from `=` to `!=`.